doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md

```diff
@@ -103,37 +103,55 @@ kubectl get service
     Because we forward the port of Grafana to `127.0.0.1:3000` in this example, we set `RAY_GRAFANA_IFRAME_HOST` to `http://127.0.0.1:3000`.
   * `http://` is required.
 
-## Step 5: Collect Head Node metrics with a ServiceMonitor
+## Step 5: Collect Head Node metrics with PodMonitors
+
+Since RayService creates two services, one managed by RayCluster and the other by RayService.
```

Comment:
```suggestion
RayService creates two Kubernetes services for the head Pod—one managed by the RayService and the other by the underlying RayCluster. Therefore, it is recommended to use a PodMonitor to monitor the metrics for head Pods to avoid misconfigurations that could result in double counting the same metrics when using a ServiceMonitor.
```